charts/prefect-agent/templates/NOTES.txt

```diff
@@ -1 +1 @@
-1. Check Prefect agent connections in the prefect UI at {{ .Values.config.apiUrl }}
+1. Check Prefect agent connections in the prefect UI at {{ .Values.agent.config.apiUrl }}
```

Comment:
Doesn't need to be in this PR, but should we have a more comprehensive notes file, since this is shown whenever `helm install` or `helm upgrade` finishes?